src/modules/material-requests/services/material-requests.service.ts

```diff
@@ -204,16 +204,17 @@ export class MaterialRequestsService {
 		userProfileId: string,
 		materialRequestDto: Pick<
 			App_Material_Requests_Set_Input,
-			'type' | 'reason' | 'organisation' | 'requester_capacity'
+			'type' | 'reason' | 'organisation' | 'requester_capacity' | 'is_pending'
 		>
 	): Promise<MaterialRequest> {
-		const { type, reason, organisation, requester_capacity } = materialRequestDto;
+		const { type, reason, organisation, requester_capacity, is_pending } = materialRequestDto;
 
 		const updateMaterialRequest = {
 			type,
 			reason,
 			organisation,
 			requester_capacity,
+			is_pending,
```

Comment:
We don't want to expose the is_pending to the api
This is set to false when you send your material requests, but we don't want anyone to be able to change this directly from the client side.